src/aiohabit/run.py

```diff
@@ -63,8 +63,8 @@ def aiohabitcli(ctx, config, db):
     """Multihost human friendly provisioner."""
     init_providers()
     ctx.ensure_object(dict)
-    ctx[DB] = init_db(db)
-    ctx[CONFIG] = init_config(config)
+    setattr(ctx, DB, init_db(db))
```

Comment:
In other project I've fixed this issue by:

```python
@click.pass_context
def managercli(ctx, config, verbose):
    """Uploads new OpenStack images from OS composes"""
    ctx.ensure_object(dict)
    ctx.obj[CONFIG] = init_config(config)
```

Reply:
So what what would be our preference?

Reply:
The method which I wrote. It is also used in the official documentation: https://click.palletsprojects.com/en/7.x/commands/#nested-handling-and-contexts

Reply:
I.e. my initial implementation in aiohabit simply forgot the `obj` part.

Reply:
Let me update it.